File: Practice/src/coding_basic_training/EnlargePicture.js
function solution(picture, k) {
    var answer = [];
    picture.map((v) => v.map((v) => v.repeat(k)));
    for (let i of picture) {
        let s = '';
        for (let j of i) {
            s += j.repeat(k);
        }
        for (let j = 0; j < k; j++) {
            answer.push(s);
        }
    }
    // picture.forEach((line) => {
    //     const expanded = [...line].reduce(
    //         (acc, cur) => acc + cur.repeat(k),
    //         ''
    //     );

    //     for (let i = 0; i < k; i++) answer.push(expanded);
    // });
    return answer;
}

File: Practice/src/coding_basic_training/EnlargePicture.java
package coding_basic_training;

public class EnlargePicture {
//	그림 확대
	public String[] solution(String[] picture, int k) {
		String[] answer = new String[picture.length * k];
		for (int i = 0; i < picture.length; i++) {
			String temp = "";
			for (char c : picture[i].toCharArray()) {
				temp += ("" + c).repeat(k);
			}
			for (int j = 0; j < k; j++) {
				answer[k * i + j] = temp;
			}
		}
		return answer;
	}
}